From c9b5bc178bce65684c04c1280d0857b3980405af Mon Sep 17 00:00:00 2001
From: Tomas Henzl <thenzl@redhat.com>
Date: Mon, 8 Sep 2014 14:31:47 -0400
Subject: [scsi] hpsa: remove unused fields from struct ctlr_info

Message-id: <1410186734-19986-57-git-send-email-thenzl@redhat.com>
Patchwork-id: 92071
O-Subject: [RHEL6.6 e-stor PATCH 56/83] hpsa: remove unused fields from struct ctlr_info
Bugzilla: 1069185
RH-Acked-by: Maurizio Lombardi <mlombard@redhat.com>
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Ewan Milne <emilne@redhat.com>

The fields "major", "max_outstanding", and "usage_count"
of struct ctlr_info were not used for anything.

Signed-off-by: Stephen M. Cameron <scameron@beardog.cce.hp.com>
Reviewed-by: Mike Miller <michael.miller@canonical.com>
Reviewed-by: Webb Scales <webb.scales@hp.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
(cherry picked from commit 84ce1ee5bf0de5daec3d28d009d74099cee3cf46)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/hpsa.c b/drivers/scsi/hpsa.c
index ae9b6c9..1c21ad5 100644
--- a/drivers/scsi/hpsa.c
+++ b/drivers/scsi/hpsa.c
@@ -5494,8 +5494,6 @@ static void start_io(struct ctlr_info *h)
    * condition.
    */
   h->commands_outstanding++;
-  if (h->commands_outstanding > h->max_outstanding)
-   h->max_outstanding = h->commands_outstanding;
 
   /* Tell the controller execute command */
   spin_unlock_irqrestore(&h->lock, flags);
diff --git a/drivers/scsi/hpsa.h b/drivers/scsi/hpsa.h
index 1e3cf33..75414a0 100644
--- a/drivers/scsi/hpsa.h
+++ b/drivers/scsi/hpsa.h
@@ -116,11 +116,8 @@ struct ctlr_info {
  int  nr_cmds; /* Number of commands allowed on this controller */
  struct CfgTable __iomem *cfgtable;
  int interrupts_enabled;
- int major;
  int  max_commands;
  int commands_outstanding;
- int  max_outstanding; /* Debug */
- int usage_count;  /* number of opens all all minor devices */
 # define PERF_MODE_INT 0
 # define DOORBELL_INT 1
 # define SIMPLE_MODE_INT 2
-- 
1.7.1